src/shared/copy.c

```diff
@@ -794,6 +797,68 @@ static int prepare_nocow(int fdf, const char *from, int fdt, unsigned *chattr_ma
         return 0;
 }
 
+/* Copies fs-verity status.  May re-open fdt to do its job. */
+static int copy_fs_verity(int fdf, int *fdt) {
+        assert(fdf >= 0);
+        assert(fdt);
+        assert(*fdt >= 0);
+
+#ifdef FS_IOC_READ_VERITY_METADATA
```

Comment:
instead of conditioning the code here please add fallback definitions to src/basic/missing_fs.h matching the kernel definitions. We try to minimize the ifdeffery in the actual code as much as we can, and adding a bunch of ioctl numbers to our headers as a fallback if system headers don't provide sounds like the best way for us to minimize ifdeffery

Reply:
do a meson check. see how this is done in src/basic/missing_sched.h for sched_attr for example

Reply:
a better blueprint to copy from is probably "struct mount_attr" even, since it also is linux/fs.h stuff